axes/models.py

```diff
@@ -56,12 +56,8 @@ class AccessAttempt(AccessBase):
         _('Failed Logins'),
     )
 
-    @property
-    def failures(self):
-        return self.failures_since_start
-
     def __str__(self):
-        return 'Attempted Access: %s' % self.attempt_time
+        return 'Attempted Access: {}'.format(self.attempt_time)
```

Comment:
@aleksihakli fstrings maybe?

Reply:
f-strings are only available in Python 3.6+ and we're still supporting Python 3.5 and PyPy 3.5 so I opted for the old-fashion `.format`. It should be easy enough to migrate to f-strings when the time is ripe and Python 3.5 support is removed.

Reply:
@aleksihakli oh, you are right, don't know why I thought we will support >= 3.6.